.github/workflows/review_pull_request.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     environment: template-compose
     steps:
       - name: Set up JDK 11
```

Comment:
Should we remove the old jdk? Since we're adding version 17 🤔

Reply:
@doannimble The old JDK 11 is to test the deprecated `template-xml`. We need to remove it [here](https://github.com/nimblehq/android-templates/pull/519#discussion_r1329484494) instead.